.github/workflows/update-submodules.yml

```diff
@@ -0,0 +1,35 @@
+name: "Update Submodules"
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sync:
+    name: "Update submodules"
+    runs-on: ubuntu-latest
```

Comment:
where can i find the syntax / definition of this file?

Reply:
here is a quick start https://docs.github.com/en/actions/quickstart#creating-your-first-workflow

some details on specifically `run-on` and jobs definitions https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#overview